src/app/wallet/send/send.component.html

```diff
@@ -174,27 +174,36 @@ <h5 fxFlex="10%"> Amount </h5>
         <md-divider></md-divider>
       </md-list>
 
-      <md-list *ngIf="send.input === 'private'">
+      <md-list *ngIf="send.input === 'anon_balance'">
         <md-list-item>
           <div class="md-line" fxFlex="100%" fxLayout.lg="row" fxlayout.lt-sm="column" fxLayout.sm="column"
                fxlayout.lt-xs="column" fxLayout.xs="column">
             <h5 fxFlex="10%"> Privacy level </h5>
 
             <div fxFlex="'column'" fxLayoutWrap>
+              <div class="privacy-labels" fxFlex="100%">
+                    <span fxFlex="" fxLayoutAlign="start center" class="privacy-label cursor-pointer"
+                          (click)="progress = 10; setPrivacy(2)">Low</span>
```

Comment:
I don't think low should ever be less than 3, infact I think the transaction will be rejected if its less than 3

Reply:
you could even set low to 4

Reply:
Maybe create one function, setPrivacy, which will set progress and set the level? Instead of doing 2 operations in the html...